Packaging.props

```diff
@@ -2,14 +2,14 @@
   <PropertyGroup>
     <PreReleaseLabel>preview2</PreReleaseLabel>
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
-    <PackageLicenseFile>$(ProjectDir)LICENSE</PackageLicenseFile>
-    <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
+    <PackageLicenseFile>$(ProjectDir)LICENSE.TXT</PackageLicenseFile>
```

Comment:
I suggested that to @tarekgh. For two reasons:

- We should standardize these key files across our repos.
- Our text files should open on double click when they are included in a distribution (.nupkg, msi, ...).